sql-db/vertx-sql/pom.xml

```diff
@@ -79,6 +87,20 @@
         </dependency>
     </dependencies>
     <profiles>
+        <profile>
```

Comment:
I don't like the idea of disabling native for all tests. Can we do the opposite, and include oracle dependencies only when native is disabled?

Reply:
Not really, I need Oracle dependencies in the code. What I can do is remove oracle from `vertx-sql` and create a new module `vertx-sql-oracle`, which would be analogous, but dedicated just to Oracle, and have native disabled. And we could merge those two modules once the issue with native build is resolved, similarily to what was done in https://github.com/quarkus-qe/quarkus-test-suite/pull/434.

Reply:
@rsvoboda WDYT?
I see 3 options:

1. Drop Oracle form `vertx-sql` and create a clone module just for Oracle.
2. Drop Oracle from `vertx-sql` with no substitution.
3. Disable native in `vertx-sql`.

Reply:
The native issue has been resolved (sice 2.10.0.Final). I think we can proceed with this PR. The remaining unresolved issues are listed in the PR description.